static/main.js

```diff
@@ -539,6 +540,28 @@ class Game {
             this.nextKey = data.nextKey;
         }
 
+        if ("timetable" in data) {
+            this.timetable = data.timetable;
+            this.timetableDepth = this.timetable.length;
+            this.turnsCount = this.timetableDepth - 1;
```

Comment:
```suggestion
            this.turnsCount = this.timetableDepth - 2;
```
*Объяснение.* Я запустил игру с 13 клиентами. И в самом начале у клиента № i (3 < i < 12) высвечивается "Ты будешь угадывать через ${i-3} ходов" (всё верно), а у клиентов № 12 и 13 - "Ты будешь угадывать через >9 ходов", что уже неверно для клиента № 12 (у него-то достигается равенство).

Reply:
Да, ты прав.